zone/lua_mod.h

```diff
@@ -27,6 +28,7 @@ class LuaMod
 	void GetEXPForLevel(Client *self, uint16 level, uint32 &returnValue, bool &ignoreDefault);
 	void GetExperienceForKill(Client *self, Mob *against, uint64 &returnValue, bool &ignoreDefault);
 	void CalcSpellEffectValue_formula(Mob *self, uint32 formula, int64 base_value, int64 max_value, int caster_level, uint16 spell_id, int ticsremaining, int64 &returnValue, bool &ignoreDefault);
+	void RegisterBug(Client *self, BaseBugReportsRepository::BugReports bug, bool &ignoreDefault);
```

Comment:
Any reason to use the Base repository specifically?

Can we also use `snake_case` here?

Reply:
I used Base Repository mainly just because it combined all the data needed, and I wouldn't need to define a new struct just to pass it again. But if there's a suggested alternative, I'm happy to adjust.

 use snakecase where? bug is a single word, self too, and ignoreDefault is the pattern across all otehr lua mod params.

Reply:
> I used Base Repository mainly just because it combined all the data needed, and I wouldn't need to define a new struct just to pass it again. But if there's a suggested alternative, I'm happy to adjust.
> 
> use snakecase where? bug is a single word, self too, and ignoreDefault is the pattern across all otehr lua mod params.

Just didn't know if it was purposeful to use Base or not.

And yeah, I was just talking about `ignoreDefault`, we can leave it as is if we prefer.

Reply:
If we change ignoreDefault, it may make sense to change it on all functions listed.